app/controllers/topical_events_controller.rb

```diff
@@ -18,4 +18,12 @@ def show
       end
     end
   end
+
+  def array_of_links_to_organisations(organisations)
+    organisations.map do |organisation|
+      helpers.link_to organisation[:title], organisation[:base_path], class: "organisation-link govuk-link"
```

Comment:
It should be possible to just use `link_to` instead of `helpers.link_to`.

Reply:
This doesn't work unfortunately, normally helpers methods are only expected to be used in the views, but the [Helpers Proxy](https://api.rubyonrails.org/v5.1/classes/ActionController/Helpers.html) is a way around that.